.github/workflows/build-and-deploy.yml

```diff
@@ -142,6 +142,50 @@ jobs:
       - name: Test package
         shell: bash
         run: ./gradlew testJpackageOutput --info     
+  # Test PKGBUILD for aur release
+  test-aur-package:
+    needs: 
+      - select-java-version
+      - build
+    runs-on:  ubuntu-22.04
+    container: 
+      image: archlinux:base-devel
+    if: >-
+      github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
why? This should run every time

Reply:
Ok, I used the configuration of "test-packages". Should I remove the condition in that too?

Reply:
I think so. This check is meant to avoid running the step if there are no secrets. If the step does not requires secrets, it may as well run